mofapy2/notebooks/getting_started_python.ipynb

```diff
@@ -16,7 +16,7 @@
    "metadata": {},
    "source": [
     "This notebook contains a detailed tutorial on how to train MOFA using Python.\n",
-    "A template script to run the code below can be found [here](https://github.com/bioFAM/MOFA2/blob/master/template_script.py)"
+    "A template script to run the code below can be found [here](https://github.com/bioFAM/MOFA2/blob/master/template_script.py)."
```

Comment:
Note that this line refers to a dead link.

Reply:
Might be this one [https://github.com/bioFAM/MOFA2/blob/master/inst/scripts/template_script.py](https://github.com/bioFAM/MOFA2/blob/master/inst/scripts/template_script.py)